Data_Dictionary.md

```diff
@@ -211,4 +212,44 @@ LAMP calculated dataset containing planned `route_id` and `service_id` combinati
 | trip_id | string | `trip_id` from [trips.txt](https://gtfs.org/schedule/reference/#tripstxt)
 | direction_id | int8 | `direction_id` from [trips.txt](https://gtfs.org/schedule/reference/#tripstxt)
 | block_id | string | `block_id` from [trips.txt](https://gtfs.org/schedule/reference/#tripstxt)
-| static_version_key | int64 | key used to link GTFS static schedule versions between tables |
\ No newline at end of file
+| static_version_key | int64 | key used to link GTFS static schedule versions between tables |
+
+
+# GTFS Alerts Archive
+
+Each row represents an update to an Realtime Alert, paired with unique Route, Stop, and Direction information. All timestamp fields are in POSIX time, the integer number of seconds since 1 January 1970 00:00:00 UTC. All datetimes in are Eastern Standard Time.
+
+| field name | type | description |
+| ---------- | ---- | ----------- |
+| id | int64 | Unique identifier for this Alert. Subsequent updates to it will have the same ID. |
+| cause | string | String from [Cause](https://github.com/google/transit/blob/master/gtfs-realtime/spec/en/reference.md#enum-cause) enum. Will be present if `cause_detail` is not null. |
+| cause_detail | string | Description of the cause of the alert that allows for agency-specific language; more specific than the Cause. If cause_detail is included, then `cause` must also be included. |
```

Comment:
To match the format of our other data dictionaries, if a field has a direction correlation to a GTFS-RT Alerts field, it should be referenced as "equivalent to GTFS-RT `gtfs-rt-field` from (gtfs-rt reference from either gtfs.org or mbta gtfs github reference if does not exist at gtfs.org) 

No need for us to maintain our own definitions and risk getting out of sync with official sources.